control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1213,6 +1218,65 @@ func (r *reconciler) reconcileAuthOIDC(ctx context.Context, hcp *hyperv1.HostedC
 	return errors.NewAggregate(errs)
 }
 
+func (r *reconciler) reconcileAuthConfig(ctx context.Context, hcp *hyperv1.HostedControlPlane) error {
+	var errs []error
+	providerReadyStatus := metav1.ConditionTrue
+	providerReadyReason := hyperv1.AsExpectedReason
+	providerReadyMessage := "OIDC providers reconciled."
```

Comment:
"OIDC providers reconciled." sounds like this function is actually reconciling some resources, which it doesn't right now. Is there something you are planning to add later?